src/helpers/getDeckAdvice/advice/OBSIDIAN_BUTCHERS/index.js

```diff
@@ -9,27 +11,38 @@ export default cards => {
   const hasObsidianButchers = cardIds.includes('F17')
   const hasRainOfFrogs = cardIds.includes('F8')
   const hasAzureHatchers = cardIds.includes('F10')
+  const hasLudicMatriarchs = cardIds.includes('N43')
+  let hasLowLvlMatriarchs = false
+  if (hasLudicMatriarchs) {
+    hasLowLvlMatriarchs = cards.find( card =>  card.id === 'N43' ).level <= 2 ? true : false
+  }
+  const numDragons = getDragons(cards).length
   const averageLevel =
     cards.map(card => card.level).reduce((acc, level) => acc + level, 0) / 12
 
+  const dragonMsg = (hasLowLvlMatriarchs && numDragons <= 3) ? ", or more dragons to synergize with Ludic Matriarchs" : ''
```

Comment:
Let’s avoid contractions in variable names. :)
```suggestion
  const dragonMessage = (hasLowLvlMatriarchs && numDragons <= 3) ? ", or more dragons to synergize with Ludic Matriarchs" : ''
```